iscsi/iscsi.go

```diff
@@ -327,7 +327,6 @@ func (c *Connector) connectTarget(target *TargetInfo, iFace string, iscsiTranspo
 			}
 
 		}
-		return "", err
```

Comment:
This commit can also be part of one of the previous commit which touched this function 
https://github.com/kubernetes-csi/csi-lib-iscsi/pull/29/commits/e73d15e8fef2e7e2632a7ded87fd182535ceeb00